### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by aflp91**
Starting fix for GSoC broken links

### CODE REVIEW

File: announcements/_posts/2020-01-07-gsoc.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: HSF GSoC 2020 call for projects and mentors
 author: Andrei Gheata
 layout: newsletter
 symbol: glyphicon-calendar
-link: activities/gsoc.html#for-projects-and-mentors
```
Comment (agheata) [inline announcements/_posts/2020-01-07-gsoc.md]: This was a valid link (was actually working), it is the checker that is wrong...

Comment (agheata) [reply announcements/_posts/2020-01-07-gsoc.md]: But we can change it indeed to reduce the noise from the checker.

**Comment by jouvin** [comment]
The checker always had a few problems but it is not a good practice to ignore the errors. Looking at the errors, I think that most of them are real ones. Those related to https links not responding, in particular the Rucio one, reminds me a problem but I forgot the details. May be @graeme-a-stewart remembers... 

I saw one error related to a mailto URL which is not correct IMO. I have not checked the source file but I think the `mailto:` prefix may be missing.

**Comment by graeme-a-stewart** [comment]
Re. the plugin errors, there are indeed many real errors that need to be checked and cleaned up. I started to do it, but it's rather tedious and slow.

The underlying problem is that we have 1448 links that get checked so there are always spurious errors from sites that timeout or are temporarily down, so we really need to rethink this infrasctucture.

**Pull request merged**